Monika After Story/game/script-topics.rpy

```diff
@@ -12905,3 +12905,78 @@ label monika_enjoyingspring:
+    m 2esd "I mean, do you remember when she threatened to quit the club just because it was going in a direction that she didn't like?"
+    m 2rfc "Although she had her reasons, it was impulsive and very inconsiderate of her."
+    if persistent.blood_type == "B":
+        m 1wuo "Ah, and bear in mind that those are just superstitions without proof whatsoever of their validity."
+        m 1hksdrb "I know this is your blood type we are talking about, but don't mistake it for a personality test."
+        m 1eka "You're way more awesome and complex than that [player]."
+    m 2dsd "But I digress."
+    m 1eub "Next on our list, we have the O group!"
+    m 4eub "Those ones are generally very agreeable and easygoing, they are very intuitive and leading, especially when it concern other peoples."
+    m 1eud "But they are also quite unpredictable and unpunctual, sometimes even jealous or ruthless."
+    m 1lud "It does not fit perfectly, but some of the traits are shared by Sayori."
+    m 1eka "She really liked take care of other people, especially you."
+    m 4eud "Back in the days, before you even came to the club, when a fight would broke out, she'd instantly know what was the right words to calm everyone."
+    m 2dsc "But she also had many issues because of her unpunctuality."
+    if persistent._mas_pm_cares_about_dokis:
+        m 2ekd "When you know what caused it, you can't help but feeling sorry for her."
+        m 2dkc "She really cared about things, but her depression just wouldn't leave her alone."
+    else:
+        m 2tkc "I'm almost feeling sorry for the MC, having to put up with someone that unreliable."
+        m 3efd "Even if she had her reasons, she never tried to change it, to make things better. And a lot of people ended up suffering unnecessarily because of that"
+        m 1dsc "Sorry, there's no need to be angry about that, it's behind us now."
+    if persistent.blood_type == "O":
+        m 1wuo "Ah, and bear in mind that those are just superstitions without proof whatsoever of their validity."
+        m 1hksdrb "I know this is your blood type we are talking about, but don't mistake it for a personality test."
+        m 1eka "You're way more awesome and complex than that [player]."
+    m 3eub "Anyway, that leaves us with only one left."
+    m 1tsu "I saved the best for last, ehehe~."
+    m 1dua "The A group correspond to..."
+    m 1hub "Me, of course!"
+    m 4wub "Peoples that have the A type are responsible, perfectionists, wise, earnest, neat and much more."
+    m 2tsu "So there's no doubt that A type blood flow through my veins."
+    if persistent.blood_type == "A":
+        m 2eka "And it's the same as yours, [player]. We really have so much in common."
+        m 1hub "With all of those qualities, we sure do make one fine team."
+    m 1eud "Ah... Wait! I also remember reading that peoples with A type blood are also obsessive, stubborn and inconsiderate of others."
```

Comment:
I've took it on[the Wikipedia page](https://en.wikipedia.org/wiki/Blood_type_personality_theory#cite_note-AP-1), but it links to a source that I can connect to. So I can't confirm. Do you want me to remove it then?

Reply:
Do you have any other sources?

Regardless if those are accurate to the blood type, do you think Monika would call herself these things, or even hint at it?

Reply:
Yes, but this was the only one with those two traits.

I think it would be something like Monika thinking aloud and realizing that if she choose to believe it just to feel good about herself, she must also accept that those things are true as well. Since she doesn't want that, she decide to reject it all.

Reply:
There are numerous traits with each blood type, and clearly not every single one of them might fit a character. Monika can bring up the positives and feel good about herself, but that doesn't mean that she _must_ accept those specific negatives as well.